### PR TITLE
commands: add install

### PR DESCRIPTION
**Pull request by Fishrock123**
**_(To be clear - this is just `ncm details` and then a `npm install` child process.)_**

i did a thing

it took me like 30 mins because this codebase is Nice

Todo:
- [x] Ensure only our options for install are stripped, pass all the rest
- [x] Support "any installer" e.g. yarn

### CODE REVIEW

**Comment by Fishrock123** [comment]
Things to do here:
- Ensure only our options for install are stripped, pass all the rest
- Support any installer e.g. yarn
- Possibly support install without arguments (i.e. do a report before project `npm i`)

**Comment by juliangruber** [comment]
The value prop for this is that it asks whether you really want to add that package, after reviewing its details, right?

**Comment by Fishrock123** [comment]
@juliangruber correct. It’s kinda like an interactive `ncm details express@4.16.0 && npm i express@4.16.0`.

**Comment by Fishrock123** [comment]
Updated this, if folks want to 👀

File: commands/install.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+  const confirm = good ? `${COLORS.green} (Y/n)` : `${COLORS.red} (y/N)`
+
+  L(line('|➔', chalk`Install this module? {${confirm}}`, COLORS.yellow))
+  L()
+
+  const choice = (await queryReadline(chalk`{${COLORS.light1} > }`)).trim().toLowerCase()
+
+  if ((good && choice === '') || choice === 'y') {
+    const args = [getValue('installCmd'), `${name}@${version}`, ...childArgv]
```
Comment (Raynos) [inline commands/install.js]: should `installCmd` and `installBin` default to `npm install` ?

Comment (Fishrock123) [reply commands/install.js]: They already do - in `lib/config.js`. I don't think we should try to fix broken user config for this, though.

**Comment by Fishrock123** [comment]
Hmm, that parser thing is still giving us trouble.

Works:
```
@nestjs/platform-fastify @ 6.0.4
```

Does not work:
```
@nestjs/platform-fastify@6.0.4
```

**Comment by Fishrock123** [comment]
Filed a new issue for that: https://github.com/nodesource/ncm-cli/issues/131

**Pull request merged**